### PR TITLE
Fix issue: Cannot publish tweet if it is too long

### LINKED ISSUE #40: Tweet cannot be published if it is too long
**Describe the bug**
If the generated tweet is too long it cannot be published and we receive an error from Twitter API.

**To Reproduce**
1. Generate a Content with long title and description
2. Try to publish it via Github API
3. See error

**Expected behavior**
The description of the tweet should be truncated to fit in the Tweet size. Ideally only the description should be truncated

**Screenshots**
None

**Additional context**
Maybe add some tests to replicate this issue

### PR DESCRIPTION
**Pull request by Shubhcoder**
### Changes in brief:-
1. Added a function to check for content to make sure it is within limit(truncates description, if required).
2. Added test for above function.

**Note** --> *I am new to both open source and github community. So, I will be happy to receive feedbacks.*

### CODE REVIEW

File: publisher/twitter/publisher_test.go [hunk 1]
```diff
@@ -38,3 +38,24 @@ func TestPublishContentInSafeMode(t *testing.T) {
...
+	ti, s, u := "Lorem Ipsum", subtitle, "https://loremipsum.io/generator/?n=3&t=s"
+	extraData := []string{"50k", "Author: @unknown"}
+
+	cont := &domain.Content{Title: &ti, Subtitle: &s, URL: &u, ExtraData: extraData}
+
+	resp := p.prepareTweet(cont)
+
+	if len(resp) > TweetLength {
+		t.Errorf("Tweet length is %v, which is greater than %v", len(resp), TweetLength)
+	}
```
Comment (ezeoleaf) [inline publisher/twitter/publisher_test.go:60]: Looks like the build is failing due to this test.
It's returning 281 chars, do you mind check the calculations?

Also, can you add the starting error in lowercase? As defined in the Style Guide the errors messages should be in lowercase. You can read about it here:
https://github.com/golang/go/wiki/CodeReviewComments#error-strings

File: publisher/twitter/publisher.go [hunk 2]
```diff
@@ -42,17 +45,38 @@ func NewPublisher(accessKeys AccessKeys, cfg config.Config) Publisher {
 	return p
 }
 
-// prepareTweet convers a domain.Content in a string Tweet
+// prepareTweet converts a domain.Content in a string Tweet
 func (p Publisher) prepareTweet(content *domain.Content) string {
+    checkTweetData(content)
+
 	tweet := fmt.Sprintf("%s: %s\n%s\n%s",
```
Comment (ezeoleaf) [inline publisher/twitter/publisher.go:52]: It looks like there are some difference in the tabulation

**Comment by Shubhcoder** [comment]
Hi, @ezeoleaf 

Thank you for your feedback. I have fixed the issues as per your observations.

**Comment by ezeoleaf** [comment]
> Hi, @ezeoleaf
> 
> Thank you for your feedback. I have fixed the issues as per your observations.
> 
> Please take a look at it.

Nice work @Shubhcoder

**Pull request merged**